nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -353,7 +374,7 @@ public void unpack(final ProcessSession session, final FlowFile source, final Li
                 int fragmentCount = 0;
                 try (final TarArchiveInputStream tarIn = new TarArchiveInputStream(new BufferedInputStream(inputStream))) {
                     TarArchiveEntry tarEntry;
-                    while ((tarEntry = tarIn.getNextTarEntry()) != null) {
+                    while ((tarEntry = tarIn.getNextEntry()) != null) {
```

Comment:
Is there some documentation you can point to suggesting changing this method call is preferable?  also changing this tar one here given this is about zip changes is not ideal.  Generally fine but if there is a clear value to doing so